### ISSUE TITLE
Accessibillity and wcag  violation: filter input field form element don't have any label

### ISSUE THREAD

**Issue by QuBaR** [esvit/ng-table#1007]
The filter input form field don't have any label and are therefore violating accessibility and WCAG 2.0 standard.

To make the ngTable accessibility and WCAG compatible we need to fix we need to:
- insert attribute **aria-labelledby=”idOfSorting”** on each filter input form element.
- insert an attribute  **id=”sortingName”** on each sorting header span so that the filter input form element can point the aria-labelledby attribute to this span html-tag.

https://dequeuniversity.com/rules/axe/2.3/label?application=AxeChrome
